frontend/app/src/store/defi/aave/index.ts

```diff
@@ -54,18 +57,14 @@ export const useAaveStore = defineStore('defi/aave', () => {
     if (!get(activeModules).includes(Module.AAVE)) {
       return;
     }
-    const section = Section.DEFI_AAVE_BALANCES;
-    const currentStatus = getStatus(section);
+    const currentStatus = getStatus();
 
-    if (
-      isLoading(currentStatus) ||
-      (currentStatus === Status.LOADED && !refresh)
-    ) {
+    if (loading() || (currentStatus === Status.LOADED && !refresh)) {
```

Comment:
maybe we could use `isFirstLoad` here? something like `if (!(isFirstLoad() || refresh) || loading())`

Do you think it would make sense to create a new method for this in `useStatusUpdater`? a method that takes refresh as a parameter and performs the other checks internally, rather than repeating these conditions every single time?

Reply:
Yes, I believe we can do this